make/go/dep_golangci_lint.mk

```diff
@@ -6,19 +6,35 @@ $(call _conditional_include,$(MAKEGO)/base.mk)
 $(call _assert_var,CACHE_VERSIONS)
 $(call _assert_var,CACHE_BIN)
 
-# We want to ensure we rebuild golangci-lint every time we require a new Go minor version.
-# Otherwise, the cached version may not support the latest language features.
```

Comment:
Maybe a comment to say why we download? And the minor language thing is fine because they'll release a new version?